dicoogle/src/main/java/pt/ua/dicoogle/server/queryretrieve/CallDCMSend.java

```diff
@@ -36,6 +38,8 @@
  * @author Luís A. Bastião Silva <bastiao@ua.pt>
  */
 public class CallDCMSend {
+    private static final Logger logger = LoggerFactory.getLogger(CMoveServiceSCP.class);
```

Comment:
I don't know if this is the equivalent change from before, but the class names do not match. `CMoveSCP` instead of `CallDCMSend`.